pantheon-bundle/frontend/src/app/search.tsx

```diff
@@ -421,7 +421,10 @@ class Search extends Component<IAppState, ISearchState> {
   private buildSearchUrl() {
     let backend = "/modules.json?"
     backend += this.state.filterQuery
-    backend += "&offset=" + ((this.state.page - 1) * this.state.pageLimit) + "&limit=" + this.state.pageLimit
+    if (this.state.filterQuery.trim() !== "") {
```

Comment:
I prefer `this.state.filterQuery.trim().length > 0` but I won't hold the PR for this.